components/molecules/HighlightInput/highlight-input-form.tsx

```diff
@@ -145,21 +170,25 @@ const HighlightInputForm = ({ refreshCallback }: HighlightInputFormProps): JSX.E
 
         refreshCallback && refreshCallback();
         setBodyText("");
-        setPullRequestLink("");
+        setHighlightLink("");
         setTitle("");
         setDate(undefined);
         setIsDivFocused(false);
         setIsFormOpenMobile(false);
         toast({ description: "Highlight Posted!", title: "Success", variant: "success" });
       }
     } else {
-      toast({ description: "Please provide a valid issue or pull request link!", title: "Error", variant: "danger" });
+      toast({
+        description: "Please provide a valid pull request, issue or blog link!",
```

Comment:
> We should provide context only DEV links work for now. Perhaps even leave a comment on this in case we get a request for hashnode or medium.

Context

```suggestion
        description: "Please provide a valid pull request, issue or dev.to blog link!",
```